src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```
@@ -0,0 +1,111 @@
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement createStatement =
+                         connection.prepareStatement(
+                             "INSERT INTO manufacturers(name, country) values(?,?);",
+                             Statement.RETURN_GENERATED_KEYS)) {
+            createStatement.setString(1, manufacturer.getName());
+            createStatement.setString(2, manufacturer.getCountry());
+            ResultSet generatedKeys = createStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getLong(1);
+                manufacturer.setId(id);
+            }
+            createStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new RuntimeException("Error");
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Manufacturer get(Long id) {
+        Manufacturer manufacturer = new Manufacturer();
+        String getManufacturerRequest = "SELECT * FROM manufacturers WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getStatement =
+                         connection.prepareStatement(getManufacturerRequest)) {
+            getStatement.setLong(1, id);
+            ResultSet resultSet = getStatement.executeQuery();
+            if (resultSet.next()) {
+                Long idOfManufacturer = resultSet.getLong("id");
+                String name = resultSet.getString("name");
+                String country = resultSet.getString("country");
+                manufacturer = new Manufacturer(idOfManufacturer, name, country);
+            }
+
+        } catch (SQLException e) {
+            throw new RuntimeException("Error");
```

Create custom exception DataProcessingException and constructor with two parameters: String message and Throwable ex.
It should be extended from RuntimeException. You should rethrow this exception in catch block on dao layer.
Don't forget: Use informative messages for exceptions.